lib/profiles/TR/DISC.js

```diff
@@ -0,0 +1,17 @@
+exports.name = 'DISC';
+exports.config = {
+    status: 'DISC',
+    longStatus: 'Discontinued Draft',
+    styleSheet: 'W3C-DISC',
+    stabilityWarning: true,
+    track: 'Recommendation',
+};
+
+const base = require('../base');
+let rules = base.insertAfter(require('../TR').rules, 'sotd.supersedable', [
+    require('../../rules/structure/security-privacy'),
```

Comment:
why add that rule for a discontinued draft?

Reply:
There's **Privacy and Security** section in https://w3c.github.io/tr-design/p2021mockup/disc-6.3.12.1.html#sec-privacy-security

Reply:
Pubrules only checks the SOTD and the headers of the documents. The template is just using an existing document that happens to have a security and privacy section but it shouldn't be required for discontinued draft